contracts/astroport-liquid-pooler/schema/raw/execute.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Do we want to commit all the /raw folders?

Reply:
I think we can get them in to see what the frontend people prefer, later we can adjust?

Reply:
cc @elenamik